pages/data-providers/[data-provider-id]/deposit-dates.jsx

```diff
@@ -62,23 +66,23 @@ const DepositDates = ({
       )}
     </Card>
 
-    <Card className={styles.export} tag="section">
-      <h2>{texts.exporting.title}</h2>
-      <p>
-        {texts.exporting.description.render({
-          count: store.depositDates.depositDatesCount || '',
-        })}
-      </p>
-      <ExportButton
-        href={store.depositDates.datesUrl}
-        disabled={store.depositDates.isExportDisabled}
-      >
-        {texts.exporting.download}
-      </ExportButton>
+    <Card className={styles.crossRepositoryCheckRedirect} tag="section">
+      <Card.Title tag="h2">{texts.crossRepositoryCheck.title}</Card.Title>
+      <NumericValue
+        value={valueOrDefault(
+          store.depositDates.crossDepositLag?.possibleBonusCount,
+          'Loading...'
+        )}
+        caption="outputs match"
+        className={styles.outputsMatch}
+      />
+      <Button tag="a" href="#crossRepositoryCheck" variant="contained">
```

Comment:
As I wrote somewhere, I prefer hyphenized IDs because it's more like HTML and URL.
```suggestion
      <Button variant="contained" href="#cross-repository-check" tag="a" >
```